src/w3d/renderer/dx8renderer.cpp

```diff
@@ -582,29 +583,27 @@ void DX8TextureCategoryClass::Render()
 #endif
             }
 
+            RenderObjClass::Material_Override *moverride =
```

Comment:
this name is a bit odd. maybe `mat_override` would be better. I first read this as `move rride`

Reply:
Ok